parcels/scripts/utils.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
+import xarray as xr
+from os import path
+from netCDF4 import Dataset
+
+
+def compute_curvilinear_rotation_angles(mesh_filename, rotation_angles_filename, variables, dimensions):
+
+    if path.isfile(rotation_angles_filename) and path.getmtime(rotation_angles_filename) > path.getmtime(mesh_filename):
+        return
+
```

Comment:
Provide some `logger.info` to users that a angles file is being generated, and where it is stored. Or, if the file already exists, that nothing is being done?